persistence/persistence-api/src/main/java/org/cbioportal/persistence/MolecularDataRepository.java

```diff
@@ -3,11 +3,8 @@
 import java.util.List;
 import java.util.Map;
 
-import org.cbioportal.model.GeneMolecularAlteration;
-import org.cbioportal.model.GenericAssayMolecularAlteration;
-import org.cbioportal.model.GenesetMolecularAlteration;
+import org.cbioportal.model.*;
```

Comment:
Your only changes in here are wildcarding an import and a whitespace change. I don't think this should be included in the PR.

Reply:
Done!